src/moduler/aktivitet/visning/hjelpekomponenter/Informasjonsfelt.tsx

```diff
@@ -0,0 +1,44 @@
+import Tekstomrade from 'nav-frontend-tekstomrade';
+import { Normaltekst } from 'nav-frontend-typografi';
+import React from 'react';
+
+import HiddenIfHOC from '../../../../felles-komponenter/hidden-if/hidden-if';
+import DetaljFelt from './detalj-felt';
+
+interface Props {
+    tittel: React.ReactNode;
+    innhold?: string;
+    fullbredde?: boolean;
+    formattertTekst?: boolean;
+    beskrivelse?: boolean;
+}
+
+interface ChildProps {
+    children: string;
```

Comment:
Jeg er ikke så fan av å kalle ting for children hvis det alltid vil være en string. Hvis det heter children forventer jeg at det kan være en komponent feks.